docs/streaming.rst

```diff
@@ -13,6 +13,9 @@ To stream the data from a file or run unmodified, use the command::
 The number (4545) must be an unused TCP port above 1024. It will bind to
 this and stream the data to any connected clients.
 
+Keyword-argument options are explained on the `command-line tools <cli.html>`__
```

Comment:
Nitpicking: I'd call these command-line options.

We can also use a Sphinx cross reference to the specific section:

```rst
:ref:`cmd-serve-files`
:ref:`command reference <cmd-serve-files>`
```

The top form uses the heading text as the link, the second one lets you define the link text.

Reply:
I changed to the 2nd form